### PR TITLE
Package ocaml-lua.1.8

### PR DESCRIPTION
**Pull request by pdonadeo**

### CODE REVIEW

File: packages/ocaml-lua/ocaml-lua.1.8/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+]
+homepage: "https://pdonadeo.github.io/ocaml-lua/"
+bug-reports: "https://github.com/pdonadeo/ocaml-lua/issues"
+dev-repo: "git+https://github.com/pdonadeo/ocaml-lua.git"
+license: "MIT"
+build: [
+  [ "sh" "-c" "cd src/lua_c ; tar xf lua-5.1.5.tar.gz" ]
+  [ "sh" "-c" "cd src/lua_c/lua-5.1.5 ; patch -p1 -i ../lua.patch" ]
+  [ "sh" "-c" "cd src/lua_c ; mv lua-5.1.5 lua515" ]
+  [ "dune" "build" "@install" ]
```
Comment (kit-ty-kate) [inline packages/ocaml-lua/ocaml-lua.1.8/opam]:
```suggestion
  [ "dune" "build" "-p" name "-j" jobs ]
```
Dune shouldn't be call without `-p` in releases

File: packages/ocaml-lua/ocaml-lua.1.8/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  [ "dune" "build" "@doc" ] {with-doc}
```
Comment (kit-ty-kate) [inline packages/ocaml-lua/ocaml-lua.1.8/opam]:
```suggestion
  [ "dune" "build" "-p" name "-j" jobs "@doc" ] {with-doc}
```
Same here

File: packages/ocaml-lua/ocaml-lua.1.8/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+]
+install: [ "dune" "install" ]
```
Comment (kit-ty-kate) [inline packages/ocaml-lua/ocaml-lua.1.8/opam]:
```suggestion
```
This is not necessary, dune already generates a `<pkg>.install` file that is going to be used by opam

File: packages/ocaml-lua/ocaml-lua.1.8/opam [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+remove: [[ "dune" "uninstall" ]]
```
Comment (kit-ty-kate) [inline packages/ocaml-lua/ocaml-lua.1.8/opam]:
```suggestion
```
Not necessary either

File: packages/ocaml-lua/ocaml-lua.1.8/opam [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/ocaml-lua/ocaml-lua.1.8/opam]:
```suggestion
  "dune" {>= "1.11"}
```
Dune shouldn't be tagged as a build dependency (see #14266). Furthermore the `dune-project` file in the source repository says this package requires at least dune 1.11

File: packages/ocaml-lua/ocaml-lua.1.8/opam [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+Lua is a powerful, light-weight programming language designed for
+extending applications. It provides a good general purpose programming
+language to replace DSL that don't really need to be specific.
+
+This library provides bindings to Lua API which allows the application
+to exchange data with Lua programs and also to extend Lua with OCaml
+functions.
+
+[Lua homepage](http://www.lua.org)"""
+flags: light-uninstall
```
Comment (kit-ty-kate) [inline packages/ocaml-lua/ocaml-lua.1.8/opam]:
```suggestion
```

**Comment by pdonadeo** [comment]
@kit-ty-kate thanks for all your suggestions: you could have simply reject the PR and instead you decided to spend time to teach something.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**